### PR TITLE
Missing indexes

### PR DESCRIPTION
**Pull request by SimonCropp**
fixes https://github.com/Particular/NServiceBus.Persistence.Sql/issues/54

### CODE REVIEW

**Comment by SimonCropp** [comment]
@Particular/sqlpersistence-maintainers review please

File: src/ScriptBuilder/Outbox/Create_MsSqlServer.sql [hunk 1]
```diff
@@ -18,4 +18,37 @@ set @createTable = '
...
 exec(@createTable);
+end
+
+if not exists
+(
+    select *
+    from sys.indexes
+    where
+        name = 'Index_DispatchedAt' and
+        object_id = object_id(@tableName)
```
Comment (SzymonPobiega) [inline src/ScriptBuilder/Outbox/Create_MsSqlServer.sql]: I know that this is not directly related to this PR but you can build `@tableName` using `QUOTENAME` instead of manually adding `[` and `]`

**Comment by SzymonPobiega** [comment]
LGTM

File: src/ScriptBuilder/Outbox/Create_MsSqlServer.sql (hunk 1 above)
Comment (SimonCropp) [inline src/ScriptBuilder/Outbox/Create_MsSqlServer.sql]: @SzymonPobiega lets do a QUOTENAME PR after this?

**Comment by DavidBoike** [comment]
Looks good to me as well. I'll retest with an unstable package from TeamCity and speak up if I see anything out of sorts. So no news is good news. Thanks @SimonCropp.
